src/main/kotlin/day3/Hero.kt

```
@@ -0,0 +1,19 @@
+package day3
+
+class Hero(
+    var name: String,
+    var maxHp: Int = 100,
+    var hp: Int = maxHp,
+) {
+    fun attack() {}
```

detekt (1.23.7)

[warning] This empty block of code can be removed.

(detekt.empty-blocks.EmptyFunctionBlock)